src/components/Molecules/SingleMessage/SingleMessage.test.js

```diff
@@ -68,6 +68,21 @@ it('renders Single Message with full width correctly', () => {
   expect(tree).toMatchSnapshot();
 });
 
+it('renders Single Message with full width image and no text correctly', () => {
+  const tree = renderWithTheme(
+    <SingleMessage
+      backgroundColor="purple"
+      imageSet={data.images}
+      image={data.image}
+      imageAltText=""
+      copyFirst={false}
+      fullImage
+    ></SingleMessage>
```

Comment:
I think we don't need to add `</SingleMessage>`

Reply:
indeed we don't